<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>471</width>
      <height>205</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="3">
      <widget class="QPushButton" name="buttonSum">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="4">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="display1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="display2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="displayResult"/>
       </item>
      </layout>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="button9">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="buttonClear">
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="buttonEnter">
       <property name="text">
        <string>enter</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="buttonDifference">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="button0">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="button3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="button7">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="button1">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="buttonDivide">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="button2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="button4">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="button6">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="button8">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="buttonMultiply">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="button5">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Number 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Number 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Result</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
